uservice_ccutter/plugins/load_plugin.py

```diff
@@ -0,0 +1,30 @@
+"""Load a plugin module by name"""
+import importlib
+import sys
+from apikit import BackendError
+try:
+    from importlib import ModuleNotFoundError
+except ImportError:
+    ModuleNotFoundError = ImportError
+
+
+def load_plugin(plugin_name):
+    """Load a named plugin.  Substitute "-" with "_" first.  If the module
+    is already loaded, return it; if not, load it and then return the
+    loaded module."""
+    ctt = "." + plugin_name.lower().replace("-", "_")
+    ctr = ".".join(__name__.split(".")[:-1])
+    modname = ctr + ctt
+    mod = None
```

Comment:
Could you `return mod` from inside the `try`, thereby obviating needing the `mod = None` line?